src/components/FacetHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_faceting.scss';
+
+import { useRef, useState } from 'react';
+
+// Components
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+
+type FacetHeaderProps = {
+  key: string;
```

Comment:
I think you were asking about this one and I didn't fully understand what you meant before. I was looking at where this `id` is used. We're using this in testing to get the button that we want to click on. Given that I've moved the click target to be on the `Accordion.Header`,  we should also make sure the `id` is attached to the same element.

So you don't need to pass this `key` here and the id should be defined on the `Accordion.Header`.

Reply:
Resolved